idl/github.com/uber/cadence/shared.thrift

```diff
@@ -1468,6 +1468,24 @@ struct DescribeTaskListResponse {
   20: optional TaskListStatus taskListStatus
 }
 
+struct GetTaskListPartitionInfoResponse {
```

Comment:
IMO, ListTaskListPartitions() is a better name for this API. GetXXX is typically used to read/return info about a specific entity.